README.md

```diff
@@ -1,9 +1,81 @@
-# DTS API
+# Distributed Text Services (DTS)
 
-The DTS API original Draft (for discussions) is described in this document: [Distributed Text Services API](distributed-text-services-api.md).
+*The DTS Specification is currently in Candidate Recommendation Status.*
+
+## What
+
+The Distributed Text Services (DTS) Specification defines a Hypermedia-Driven Web API for working with collections of text as machine-actionable data.
+It specifies 3 distinct operation endpoints:
+
+- The [Collection Endpoint](Collection-Endpoint.md) is used to navigate the text collection contents 
+- The [Navigation Endpoint](Navigation-Endpoint.md) is used to navigate within a single text document
+- The [Document Endpoint](Document-Endpoint.md) is used to retrieve complete or partial texts
+
+The Collection and Navigation endpoints return JSON adhering to the [W3C Hydra standard](http://www.hydra-cg.com/spec/latest/core/). The Document returns TEI/XML of the requested text or fragment.
```

Comment:
Add something about Restfulness ?

Reply:
RESTfulness is a quagmire I don't really want to go into. REST purists would say our API is not RESTful because the urls don't identify resources.